src/Microsoft.AspNetCore.Dispatcher/DispatcherLoggerExtensions.cs

```diff
@@ -31,9 +31,9 @@ public static void AmbiguousEndpoints(this ILogger logger, string ambiguousEndpo
             _ambiguousEndpoints(logger, ambiguousEndpoints, null);
         }
 
-        public static void EndpointMatched(this ILogger logger, string endpointName)
+        public static void EndpointMatched(this ILogger logger, Endpoint endpoint)
```

Comment:
@jbagga - this is something I usually do with these kinds of logging methods (pass objects rather than strings). This way all of the complexity is hidden behind this method instead of the surrounding code.